offline_signing/transfer.sh

```diff
@@ -6,6 +6,11 @@ declare -A APPID
 APPID[amd64-usr]=e96281a6-d1af-4bde-9a0a-97b76e56dc57
 APPID[arm64-usr]=103867da-e3a2-4c92-b0b3-7fbd7f7d8b71
 
+declare -A RELEASE_CHANNEL
```

Comment:
If we're defining these channel names, would it make sense to just map the canary channels as well?  Is there a reason we'd need to change them through those environment variables?

Reply:
That was done solely to avoid publishing the channel names.  The variable assignments can go into `~/.config/roller.conf` and the internal release docs will specify their values.

Reply:
Okay, I forgot those aren't published.  Makes sense then.